react-function-components/src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+
+const element = CustomButton();
+root.render(element);
```

Comment:
In general, we don't call our own components in React apps. Instead, create a react element

```jsx
const element = <CustomButton />;
root.render(element);
```